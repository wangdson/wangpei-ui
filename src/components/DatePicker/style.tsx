import styled, { css } from 'styled-components';
import { modalCloseAnimate, modalOpenAnimate } from '../shared/animation';
import { color, typography } from "../shared/style";
import { DateItem } from './index';

export const CalendarWrapper = styled.div<{visible: boolean, delay: number}> `
  position: absolute;
  border: 1px solid black;
  transition: all ${(props) => props.delay / 1000 }s cubic-bezier(0.23, 1, 0.32, 1);
  ${(props) => props.visible && css`animation: ${modalOpenAnimate} ${props.delay/1000}s ease-in;`  }
  ${(props) => !props.visible && css`animation: ${modalCloseAnimate} ${props.delay/1000}s ease-in;` }
`;

export const CalendarDateRow = styled.tr``;

export const tableItemStyle = css`
	display: inline-block;
	min-width: 24px;
	height: 24px;
	line-height: 24px;
	border-radius: 2px;
	margin: 2px;
	text-align: center;
`;

export const CalendarHeadItem = styled.td`
  ${tableItemStyle}
  cursor:default;
`;

export const CalendarHeadWrapper = styled.div`
  padding: 10px;
  display: flex;
  color: rgba(0, 0, 0, 0.85);
  border-bottom: 1px solid #f0f0f0;
  justify-content: center;
`;

export const btnStyle = {
	padding: "0px",
	background: color.lightest,
};

export const IconWrapper = styled.span`
  display: inline-block;
  vertical-align: middle;
  > svg {
    height: 12px;
  }
`;

export const BtnDiv = styled.div`
  display: flex;
  jutify-content: center;
  align-items: center;
  height: 24px;
  line-height: 24px;
`;

export const CalendarDate = styled.td<Partial<DateItem>>`
  display: inline-block;
  min-width: 24px;
  height: 24px;
  line-height: 24px;
  border-radius: 2px;
  margin: 2px;
  text-align: center;
  cursor: pointer;
  ${(props) => {
    if (props.isToday) {
      return `color:${color.lightest};background:${color.secondary};`;
    }
  }}
  ${(props) => {
    if (props.isonDay) {
      //当天的 
      return `color:${color.lightest};background:${color.primary};`;
    }
    return `&:hover {color: ${color.secondary};};`;
  }}
  ${(props) => {
    if (props.isonMonth === false) {
      //不是当月显示灰色
      return `color:${color.mediumdark};`;
    }
    return "";
  }}
`;

export const MonthWrapper = styled.div`
  flex-wrap: wrap;
  justify-content: center;
  align-items: center;
  position: relative;
`;

export const MonthItem = styled.div<{ toGrey?: boolean }>`
  width: 25%;
  height: 60px;
  display: flex;
  justify-content: center;
  align-items: center;
  cursor: pointer;
  ${(props) => props.toGrey && `color:${color.mediumdark};`}
  &:hover {
    color: ${color.secondary};
  }
`;

export const Bwrapper = styled.b`
  cursor: pointer;
  &:hover {
    color: ${color.primary};
  }
`;

export const CalendarIcon = styled.span`
  display: inline-block;
`; 

export const DatePickerWrapper = styled.div`
  display: inline-block;
  border-color: #40a9ff;
  border-right-width: 1px !important;
  outline: 0;
  box-shadow: 0 0 0 2px rgba(24, 144, 255, 0.2);
`;
