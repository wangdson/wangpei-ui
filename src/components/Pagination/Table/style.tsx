import styled from 'styled-components';

export const TableTable = styled.table`
width: 100%;
text-align: left;
border-radius: 2px 2px 0 0;
border-collapse: separate;
border-spacing: 0;
table-layout: auto;
box-sizing: border-box;
margin: 0;
padding: 0;
color: rgba(0, 0, 0, 0.65);
font-variant: tabular-nums;
line-height: 1.5715;
list-style: none;
font-feature-settings: "tnum";
position: relative;
z-index: 0;
clear: both;
font-size: 14px;
background: #fff;
border-radius: 2px;
& > thead > tr > th {
  color: rgba(0, 0, 0, 0.85);
  font-weight: 500;
  text-align: left;
  background: #fafafa;
  border-bottom: 1px solid #f0f0f0;
  transition: background 0.3s ease;
  position: relative;
  padding: 16px;
  overflow-wrap: break-word;
}
& > tbody > tr {
  & > td {
    border-bottom: 1px solid #f0f0f0;
    transition: background 0.3s;
    position: relative;
    padding: 16px;
    overflow-wrap: break-word;
  }
  &:hover {
    & > td {
      background: #fafafa;
    }
  }
}
`;

export const TableHeadSpan = styled.span`
  display: inline-block;
  position: absolute;
  right: 0;
  top: 0;
  padding: 4px 16px 16px;
  cursor: pointer;
  & svg {
    height: 10px;
    width: 10px;
  }
`;