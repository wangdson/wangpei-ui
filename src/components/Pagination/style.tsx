import styled from 'styled-components';

export const PageUl = styled.ul`
  display: flex;
  justify-content: center;
  align-items: center;

  & > li {
    list-style: none;
    margin: 4px;
  }
  & button {
    border-radius: 6px;
    padding: 10px 8px;
  }
  & span {
    line-height: 13.6px;
    height: 13.6px;
    min-width: 18px;
  }
  & svg {
    height: 13.6px;
    width: 13.6px;
    vertical-align: bottom;
  }
`;