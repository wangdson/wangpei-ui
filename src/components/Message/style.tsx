import styled, {css} from 'styled-components';
import { typography } from "../shared/style";
import { iconSpin, messageOpenAnimate, messageCloseAnimate } from '../shared/animation';

export const messageBoxShadow = css`
	box-shadow: 0 3px 6px -4px rgba(0, 0, 0, 0.12),
		0 6px 16px 0 rgba(0, 0, 0, 0.08), 0 9px 28px 8px rgba(0, 0, 0, 0.05);
`;

export const MessageText = styled.span<{ bg: string; fc: string }>`
	display: inline-block;
	padding: 10px 16px;
	background: ${(props) => props.bg};
	color: ${(props) => props.fc};
	font-size: ${typography.size.s2}px;
	font-weight: ${typography.weight.bold};
	margin: 10px;
	${messageBoxShadow};
	border-radius: 2px;
`;

export const IconWrapper = styled.span<{ spin?: boolean }>`
	margin-right: 10px;
	& > svg {
		font-size: ${typography.size.s2}px;
		${(props) =>
			props.spin &&
			css`
				animation: ${iconSpin} 2s linear infinite;
			`}
	}
`;

export const MessageTextWrapper = styled.div<{
	openState: boolean;
	closeState: boolean;
	ani: number;
}>`
	${(props) =>
		props.openState &&
		css`
			animation: ${messageOpenAnimate} ${props.ani / 1000}s ease-in;
		`}
	${(props) =>
		props.closeState &&
		css`
			animation: ${messageCloseAnimate} ${props.ani / 1000}s ease-in;
		`}
`;