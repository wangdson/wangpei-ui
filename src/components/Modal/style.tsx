import styled, {css} from 'styled-components';
import {typography} from '../../components/shared/style'; 
import { modalOpenAnimate, modalCloseAnimate } from '../../components/shared/animation'; 

export const ModalWrapper = styled.div`
	position: fixed;
	top: 0;
	left: 0;
	right: 0;
	bottom: 0;
	z-index: 1000;
`;

export const ModalViewPort = styled.div<{ visible: boolean; delay: number }>`
background: #fff;
border: none;
border-radius: 5px;
box-shadow: 2px 2px 4px #d9d9d9;
text-shadow: 1px 1px 4px #d9d9d9, -1px -1px 4px #fff;
margin: 0 auto;
min-width: 320px;
overflow: hidden;
padding: 8px;
position: relative;
top: 100px;
transition: all 0.5s cubic-bezier(0.23, 1, 0.32, 1);
width: 30%;
z-index: 1001;
${(props) =>
  props.visible &&
  css`
    animation: ${modalOpenAnimate} ${props.delay / 1000}s ease-in;
  `}
${(props) =>
  !props.visible &&
  css`
    animation: ${modalCloseAnimate} ${props.delay / 1000}s ease-in;
  `}
`;

export const ModalMask = styled.div`
  background-color: rgba(0, 0, 0, 0.45);
  bottom: 0;
  left: 0;
  position: fixed;
  right: 0;
  top: 0;
`;

export const CloseBtn = styled.div`
  position: absolute;
  right: 5px;
  top: 5px;
`;

export const ConfirmWrapper = styled.div`
	display: flex;
	justify-content: flex-end;
	align-item: center;
	padding: 10px;
`;

export const ChildrenWrapper = styled.div`
	min-height: 100px;
	padding: 10px;
`;

export const TitleWrapper = styled.div`
  height: 30px;
  line-height: 30px;
  font-size: ${typography.size.m2}px;
  font-weight: ${typography.weight.bold};
  padding: 5px;
`;