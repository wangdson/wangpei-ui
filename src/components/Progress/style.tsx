import styled,{keyframes} from 'styled-components';
import {typography} from '../shared/style';

export const progressFlash = keyframes`
  0% {
    opacity: 0.1;
    width: 0;
  }
  20% {
    opacity: 0.5;
    width: 0;
  }
  100% {
    opacity: 0;
    width: 100%
  }
`;

export const BarWrapper = styled.div`
  display: flex;
  padding: 5px;
  align-items: center;
`; 

export interface BarMainProps {
	state: number;
	height?: number;
	flashColor: string;
	primary: string;
	secondary: string;
}

export const BarMain = styled.div<BarMainProps>`
  width: ${(props) => props.state}%;
  height: ${(props) => (props.height ? props.height : 8)}px;
  background-color: ${(props) => props.primary};
  background-image: linear-gradient(
    to right,
    ${(props) => props.primary},
    ${(props) => props.secondary}
  );
  transition: all 0.4s cubic-bezier(0.08, 0.82, 0.17, 1) 0s;
  border-radius: 5px;
  position: relative;
  &::before {
    animation: ${progressFlash} 2.4s cubic-bezier(0.23, 1, 0.32, 1) infinite;
    background: ${(props) => props.flashColor};
    border-radius: 10px;
    bottom: 0;
    content: "";
    left: 0;
    opacity: 0;
    position: absolute;
    right: 0;
    top: 0;
  }
`;

export const BarMainWrapper = styled.div<{ bottomColor: string; height?: number }>`
	width: 100%;
	border-radius: 5px;
	position: relative;
	background: ${(props) => props.bottomColor};
	height: ${(props) => (props.height ? props.height : 8)}px;
`;
export const BarText = styled.div<{ height?: number }>`
	line-height: ${(props) => (props.height ? props.height : 8)}px;
	font-weight: ${typography.weight.bold};
	text-align: center;
	display: inline-block;
	margin-left: 10px;
	min-width: 55px;
`;
export const CircleWrapper = styled.div`
	position: relative;
	display: inline-block;
	border-radius: 50%;
`;

export const CircleText = styled.div<{ size: number }>`
	line-height: ${(props) => props.size * 0.62}px;
	width: ${(props) => props.size * 0.62}px;
	height: ${(props) => props.size * 0.62}px;
	border-radius: 50%;
	display: inline-block;
	font-weight: ${typography.weight.bold};
	left: 50%;
	position: absolute;
	text-align: center;
	top: 50%;
	transform: translateX(-50%) translateY(-50%);
`;


