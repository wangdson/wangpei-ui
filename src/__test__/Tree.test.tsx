import React from "react";
import { render, fireEvent, cleanup } from "@testing-library/react";
import { Tree } from "../components/Tree";
import { color, typography } from "../components/shared/style";

const source = [
	{
	  value: "北京分行",
	  children: [
		{
		  value: "朝阳支行办事处",
		  children: [
			{   value: "朝阳支行办事处-1" },
			{   value: "朝阳支行办事处-2" }
		  ]
		},
		{   value: "海淀支行办事处" },
		{  value: "石景山支行办事处" }
	  ]
	},
	{
	  value: "天津分行",
	  children: [
		{   value: "和平支行办事处" },
		{  value: "河东支行办事处" },
		{ value: "南开支行办事处" }
	  ]
	}
];

describe('test Tree component', ()=>{
  it(' test correct Tree ', ()=>{
    const wrapper = render(<Tree source={source}/>);
    expect(wrapper).toMatchSnapshot();
  });
})