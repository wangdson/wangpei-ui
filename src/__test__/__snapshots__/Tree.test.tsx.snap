// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Tree component  test correct Tree  1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div>
        <div
          class="sc-bdnylx fseIDF"
          draggable="true"
          style="padding-left: 10px; cursor: pointer;"
        >
          <span
            class="sc-gtssRu bhYnip"
          >
            <svg
              class="sc-hKFyIo epBBzH"
              data-testid="icon-svg"
              height="20px"
              viewBox="0 0 1024 1024"
              width="20px"
            >
              <path
                class="sc-eCApGN ijoIcd"
                color="black"
                d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
                data-testid="icon-path"
              />
            </svg>
          </span>
          北京分行
        </div>
        <div
          class="sc-bdnylx kzZalT"
          draggable="true"
          style="padding-left: 20px; cursor: pointer;"
        >
          <span
            class="sc-gtssRu bhYnip"
          >
            <svg
              class="sc-hKFyIo epBBzH"
              data-testid="icon-svg"
              height="20px"
              viewBox="0 0 1024 1024"
              width="20px"
            >
              <path
                class="sc-eCApGN ijoIcd"
                color="black"
                d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
                data-testid="icon-path"
              />
            </svg>
          </span>
          朝阳支行办事处
        </div>
        <div
          class="sc-bdnylx eovGvC"
          draggable="true"
          style="padding-left: 30px; cursor: pointer;"
        >
          <span
            class="sc-gtssRu bIyNgL"
          >
            <svg
              class="sc-hKFyIo epBBzH"
              data-testid="icon-svg"
              height="20px"
              viewBox="0 0 1024 1024"
              width="20px"
            >
              <path
                class="sc-eCApGN ijoIcd"
                color="black"
                d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
                data-testid="icon-path"
              />
            </svg>
          </span>
          朝阳支行办事处-1
        </div>
        <div
          class="sc-bdnylx eovGvC"
          draggable="true"
          style="padding-left: 30px; cursor: pointer;"
        >
          <span
            class="sc-gtssRu bIyNgL"
          >
            <svg
              class="sc-hKFyIo epBBzH"
              data-testid="icon-svg"
              height="20px"
              viewBox="0 0 1024 1024"
              width="20px"
            >
              <path
                class="sc-eCApGN ijoIcd"
                color="black"
                d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
                data-testid="icon-path"
              />
            </svg>
          </span>
          朝阳支行办事处-2
        </div>
        <div
          class="sc-bdnylx kzZalT"
          draggable="true"
          style="padding-left: 20px; cursor: pointer;"
        >
          <span
            class="sc-gtssRu bIyNgL"
          >
            <svg
              class="sc-hKFyIo epBBzH"
              data-testid="icon-svg"
              height="20px"
              viewBox="0 0 1024 1024"
              width="20px"
            >
              <path
                class="sc-eCApGN ijoIcd"
                color="black"
                d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
                data-testid="icon-path"
              />
            </svg>
          </span>
          海淀支行办事处
        </div>
        <div
          class="sc-bdnylx kzZalT"
          draggable="true"
          style="padding-left: 20px; cursor: pointer;"
        >
          <span
            class="sc-gtssRu bIyNgL"
          >
            <svg
              class="sc-hKFyIo epBBzH"
              data-testid="icon-svg"
              height="20px"
              viewBox="0 0 1024 1024"
              width="20px"
            >
              <path
                class="sc-eCApGN ijoIcd"
                color="black"
                d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
                data-testid="icon-path"
              />
            </svg>
          </span>
          石景山支行办事处
        </div>
        <div
          class="sc-bdnylx fseIDF"
          draggable="true"
          style="padding-left: 10px; cursor: pointer;"
        >
          <span
            class="sc-gtssRu bhYnip"
          >
            <svg
              class="sc-hKFyIo epBBzH"
              data-testid="icon-svg"
              height="20px"
              viewBox="0 0 1024 1024"
              width="20px"
            >
              <path
                class="sc-eCApGN ijoIcd"
                color="black"
                d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
                data-testid="icon-path"
              />
            </svg>
          </span>
          天津分行
        </div>
        <div
          class="sc-bdnylx kzZalT"
          draggable="true"
          style="padding-left: 20px; cursor: pointer;"
        >
          <span
            class="sc-gtssRu bIyNgL"
          >
            <svg
              class="sc-hKFyIo epBBzH"
              data-testid="icon-svg"
              height="20px"
              viewBox="0 0 1024 1024"
              width="20px"
            >
              <path
                class="sc-eCApGN ijoIcd"
                color="black"
                d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
                data-testid="icon-path"
              />
            </svg>
          </span>
          和平支行办事处
        </div>
        <div
          class="sc-bdnylx kzZalT"
          draggable="true"
          style="padding-left: 20px; cursor: pointer;"
        >
          <span
            class="sc-gtssRu bIyNgL"
          >
            <svg
              class="sc-hKFyIo epBBzH"
              data-testid="icon-svg"
              height="20px"
              viewBox="0 0 1024 1024"
              width="20px"
            >
              <path
                class="sc-eCApGN ijoIcd"
                color="black"
                d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
                data-testid="icon-path"
              />
            </svg>
          </span>
          河东支行办事处
        </div>
        <div
          class="sc-bdnylx kzZalT"
          draggable="true"
          style="padding-left: 20px; cursor: pointer;"
        >
          <span
            class="sc-gtssRu bIyNgL"
          >
            <svg
              class="sc-hKFyIo epBBzH"
              data-testid="icon-svg"
              height="20px"
              viewBox="0 0 1024 1024"
              width="20px"
            >
              <path
                class="sc-eCApGN ijoIcd"
                color="black"
                d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
                data-testid="icon-path"
              />
            </svg>
          </span>
          南开支行办事处
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div>
      <div
        class="sc-bdnylx fseIDF"
        draggable="true"
        style="padding-left: 10px; cursor: pointer;"
      >
        <span
          class="sc-gtssRu bhYnip"
        >
          <svg
            class="sc-hKFyIo epBBzH"
            data-testid="icon-svg"
            height="20px"
            viewBox="0 0 1024 1024"
            width="20px"
          >
            <path
              class="sc-eCApGN ijoIcd"
              color="black"
              d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
              data-testid="icon-path"
            />
          </svg>
        </span>
        北京分行
      </div>
      <div
        class="sc-bdnylx kzZalT"
        draggable="true"
        style="padding-left: 20px; cursor: pointer;"
      >
        <span
          class="sc-gtssRu bhYnip"
        >
          <svg
            class="sc-hKFyIo epBBzH"
            data-testid="icon-svg"
            height="20px"
            viewBox="0 0 1024 1024"
            width="20px"
          >
            <path
              class="sc-eCApGN ijoIcd"
              color="black"
              d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
              data-testid="icon-path"
            />
          </svg>
        </span>
        朝阳支行办事处
      </div>
      <div
        class="sc-bdnylx eovGvC"
        draggable="true"
        style="padding-left: 30px; cursor: pointer;"
      >
        <span
          class="sc-gtssRu bIyNgL"
        >
          <svg
            class="sc-hKFyIo epBBzH"
            data-testid="icon-svg"
            height="20px"
            viewBox="0 0 1024 1024"
            width="20px"
          >
            <path
              class="sc-eCApGN ijoIcd"
              color="black"
              d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
              data-testid="icon-path"
            />
          </svg>
        </span>
        朝阳支行办事处-1
      </div>
      <div
        class="sc-bdnylx eovGvC"
        draggable="true"
        style="padding-left: 30px; cursor: pointer;"
      >
        <span
          class="sc-gtssRu bIyNgL"
        >
          <svg
            class="sc-hKFyIo epBBzH"
            data-testid="icon-svg"
            height="20px"
            viewBox="0 0 1024 1024"
            width="20px"
          >
            <path
              class="sc-eCApGN ijoIcd"
              color="black"
              d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
              data-testid="icon-path"
            />
          </svg>
        </span>
        朝阳支行办事处-2
      </div>
      <div
        class="sc-bdnylx kzZalT"
        draggable="true"
        style="padding-left: 20px; cursor: pointer;"
      >
        <span
          class="sc-gtssRu bIyNgL"
        >
          <svg
            class="sc-hKFyIo epBBzH"
            data-testid="icon-svg"
            height="20px"
            viewBox="0 0 1024 1024"
            width="20px"
          >
            <path
              class="sc-eCApGN ijoIcd"
              color="black"
              d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
              data-testid="icon-path"
            />
          </svg>
        </span>
        海淀支行办事处
      </div>
      <div
        class="sc-bdnylx kzZalT"
        draggable="true"
        style="padding-left: 20px; cursor: pointer;"
      >
        <span
          class="sc-gtssRu bIyNgL"
        >
          <svg
            class="sc-hKFyIo epBBzH"
            data-testid="icon-svg"
            height="20px"
            viewBox="0 0 1024 1024"
            width="20px"
          >
            <path
              class="sc-eCApGN ijoIcd"
              color="black"
              d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
              data-testid="icon-path"
            />
          </svg>
        </span>
        石景山支行办事处
      </div>
      <div
        class="sc-bdnylx fseIDF"
        draggable="true"
        style="padding-left: 10px; cursor: pointer;"
      >
        <span
          class="sc-gtssRu bhYnip"
        >
          <svg
            class="sc-hKFyIo epBBzH"
            data-testid="icon-svg"
            height="20px"
            viewBox="0 0 1024 1024"
            width="20px"
          >
            <path
              class="sc-eCApGN ijoIcd"
              color="black"
              d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
              data-testid="icon-path"
            />
          </svg>
        </span>
        天津分行
      </div>
      <div
        class="sc-bdnylx kzZalT"
        draggable="true"
        style="padding-left: 20px; cursor: pointer;"
      >
        <span
          class="sc-gtssRu bIyNgL"
        >
          <svg
            class="sc-hKFyIo epBBzH"
            data-testid="icon-svg"
            height="20px"
            viewBox="0 0 1024 1024"
            width="20px"
          >
            <path
              class="sc-eCApGN ijoIcd"
              color="black"
              d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
              data-testid="icon-path"
            />
          </svg>
        </span>
        和平支行办事处
      </div>
      <div
        class="sc-bdnylx kzZalT"
        draggable="true"
        style="padding-left: 20px; cursor: pointer;"
      >
        <span
          class="sc-gtssRu bIyNgL"
        >
          <svg
            class="sc-hKFyIo epBBzH"
            data-testid="icon-svg"
            height="20px"
            viewBox="0 0 1024 1024"
            width="20px"
          >
            <path
              class="sc-eCApGN ijoIcd"
              color="black"
              d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
              data-testid="icon-path"
            />
          </svg>
        </span>
        河东支行办事处
      </div>
      <div
        class="sc-bdnylx kzZalT"
        draggable="true"
        style="padding-left: 20px; cursor: pointer;"
      >
        <span
          class="sc-gtssRu bIyNgL"
        >
          <svg
            class="sc-hKFyIo epBBzH"
            data-testid="icon-svg"
            height="20px"
            viewBox="0 0 1024 1024"
            width="20px"
          >
            <path
              class="sc-eCApGN ijoIcd"
              color="black"
              d="M511.976 768.002c-10.236 0-20.47-3.904-28.282-11.712l-344.154-344.098c-15.624-15.62-15.624-40.946-0.006-56.57 15.622-15.622 40.948-15.624 56.568-0.004l315.876 315.82 315.922-315.868c15.624-15.618 40.952-15.616 56.568 0.004 15.62 15.624 15.618 40.95-0.006 56.57l-344.204 344.144c-7.81 7.81-18.046 11.714-28.282 11.714z"
              data-testid="icon-path"
            />
          </svg>
        </span>
        南开支行办事处
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
